### PR TITLE
add stream pallet api

### PR DESCRIPTION
**Pull request by hskang9**

### CODE REVIEW

**Comment by GopherJ** [comment]
does all need to be a package? Wondering if can be merge into one `payroll` etc

**Comment by GopherJ** [comment]
also CI deployment should be changed as well maybe if finally we decided to always use seperate package, I think maybe one package should be enough.

can also be named as sdk-payroll etc

**Comment by GopherJ** [comment]
we can check how acala.js is organizing their packages https://github.com/AcalaNetwork/acala.js/tree/master/packages/sdk-homa

File: packages/stream/src/queries.ts [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  const q = gql`
+  {
+    Stream(${id}) {
+    }
+  }
+`;
+  return q;
+};
+
+const streamAccountQuery = (account: string): string => {
```
Comment (i5possible) [inline packages/stream/src/queries.ts:14]: How could we distinguish the receiving streams and sending steams.

**Comment by GopherJ** [comment]
close due to no activity

**Pull request closed without merging**